src/test/resources/testenv.properties

```diff
@@ -0,0 +1,6 @@
+#browser to be used for testing
+browser = chrome
+#Base URL of the website
+baseurl = http://opensource.demo.orangehrmlive.com
+#Location of screenshots to store
+screenshotpath = D:\\Projects\\JavaPractice\\out\\screenshots
```

Comment:
How do we know which screenshot belongs to which test under this directory?

Reply:
It depends on the screenshot name which will be formed with TestcaseID.
But, in our case we are taking screenshots into the test report itself and this storing of screenshots into a folder is been ignored.